### PR TITLE
Adding  a Blog system to fury

### PR DESCRIPTION
**Pull request by skoudoro**
Adding a blog page for

- Release announcement
- News
- Update
- GSoC work
- Tutorial

### CODE REVIEW

File: docs/source/posts/2019/2019-06-19-brain-art.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Success on Brain Art Competition using FURY
+===========================================
+
+.. post:: June 19 2019
+   :author: skoudoro
+   :tags: shader
+   :category: news
+
+
+Congratulations to `David <https://github.com/dmreagan>`_ who win the `OHBM BrainArt (MELPOMENE category) <https://www.neurobureau.org/galleries/brain-art-competition-2019-2/>`_ by using DIPY and FURY!!!
```
Comment (Garyfallidis) [inline docs/source/posts/2019/2019-06-19-brain-art.rst:11]: won

File: docs/source/posts/2018/2018-11-26-release-announcement.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.. post:: November 26 2018
+   :author: skoudoro
+   :tags: fury
+   :category: release
+
+
+The FURY project is happy to announce the release of FURY 0.1.4!
+FURY is a free and open source software library for scientific visualization and 3D animations.
+
+You can support us by `adding a star <https://github.com/fury-gl/fury/stargazers>`_ on FURY github project.
```
Comment (Garyfallidis) [inline docs/source/posts/2018/2018-11-26-release-announcement.rst:13]: You can show your support by

File: docs/source/posts/2018/2018-11-26-release-announcement.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+The **major highlights** of this release are:
+
+.. include:: ../../release_notes/releasev0.1.4.rst
+    :start-after: --------------
+
+.. note:: The complete release notes is :ref:`available here <releasev0.1.4>`
```
Comment (Garyfallidis) [inline docs/source/posts/2018/2018-11-26-release-announcement.rst:20]: are available here

**Comment by skoudoro** [comment]
Indeed @Garyfallidis, this is a template, but those announcements will be diluted by actual news like GSoC update, new features or tutorials, etc...

**Pull request merged**